### PR TITLE
fix: deprecated dependencies / install warnings

### PR DESCRIPTION
**Pull request by wingkwong**
## Change Summary

- close: #6361 
- bump: `debug`, `@google-cloud/storage`, `isomorphic-dompurify`, `tedious`, `aws-sdk`, `mssql`, `@aws-sdk/client-kafka`, `httpsnippet`
- remove unnecessary dependencies: `express-graphql` , `@types/axios`, `@fingerprintjs/fingerprintjs`
- migrate `request` to `axios`
- No upstream supported version: 
  - `clickhouse` -> `uuid@3.4.0`
  - `@jm18457/kafkajs-msk-iam-authentication-mechanism` ->  `@aws-sdk/signature-v4@3.374.0` 
  - `@jm18457/kafkajs-msk-iam-authentication-mechanism` ->  `@aws-sdk/hash-node`
  -  `har-validator@5.1.5` -> `request`
  - multiple -> ` querystring@0.2.0`
  - `sqlite3` -> `@npmcli/move-file@1.1.2`
  - `@esbuild-plugins/node-modules-polyfill` -> `rollup-plugin-inject@3.0.2`
  - `nc-help` -> `snowflake-sdk` -> `string-similarity@4.0.4` 

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by sreehari2003** [comment]
> ## Change Summary
> * close: [Deprecated Dependencies #6361](https://github.com/nocodb/nocodb/issues/6361)
> * bump: `debug`, `@google-cloud/storage`, `isomorphic-dompurify`, `tedious`, `aws-sdk`, `mssql`, `@aws-sdk/client-kafka`, `httpsnippet`
> * remove unnecessary dependencies: `express-graphql` , `@types/axios`
> * No upstream supported version:
>   
>   * `clickhouse` -> `uuid@3.4.0`
>   * `@jm18457/kafkajs-msk-iam-authentication-mechanism` ->  `@aws-sdk/signature-v4@3.374.0`
>   * `@jm18457/kafkajs-msk-iam-authentication-mechanism` ->  `@aws-sdk/hash-node`
>   * `har-validator@5.1.5` -> `request`
>   * multiple -> ` querystring@0.2.0`
>   * `sqlite3` -> `@npmcli/move-file@1.1.2`
>   * `@esbuild-plugins/node-modules-polyfill` -> `rollup-plugin-inject@3.0.2`
> * TODO:
>   
>   * `nc-help` -> `string-similarity@4.0.4`
>   * `request` -> use `fetch`?
> 
> ## Change type
> * [ ]  feat: (new feature for the user, not a new feature for build script)
> * [x]  fix: (bug fix for the user, not a fix to a build script)
> * [ ]  docs: (changes to the documentation)
> * [ ]  style: (formatting, missing semi colons, etc; no production code change)
> * [ ]  refactor: (refactoring production code, eg. renaming a variable)
> * [ ]  test: (adding missing tests, refactoring tests; no production code change)
> * [ ]  chore: (updating grunt tasks etc; no production code change)
> 
> ## Test/ Verification
> Provide summary of changes.
> 
> ## Additional information / screenshots (optional)
> Anything for maintainers to be made aware of

@wingkwong  can we upgrade the ant design as well ?

**Comment by wingkwong** [comment]
@sreehari2003 I'm fine with it. Seems there is no breaking changes.

**Comment by sreehari2003** [comment]
> @sreehari2003 I'm fine with it. Seems there is no breaking changes.

i was trying to setup storybook yesterday in my free time and it didn't went well too much version conflicts, and design was the major one

**Comment by sreehari2003** [comment]
> @sreehari2003 any specific version you want to use? I tried the latest one but it seems there are lots of incompatible changes.

any version works for me , he have a few generic components now so I thought we should document it

File: packages/nocodb/src/plugins/backblaze/Backblaze.ts [hunk 1]
```diff
@@ -47,32 +47,29 @@ export default class Backblaze implements IStorageAdapterV2 {
...
-            if (err) {
-              console.log('Error', err);
+            if (err1) {
+              console.log('Error', err1);
               reject(err1);
             }
             if (data) {
               resolve(data.Location);
             }
           });
```
Comment (pranavxc) [inline packages/nocodb/src/plugins/backblaze/Backblaze.ts:68]: We can use async...await and ignore the Promise wrapping

**Review by pranavxc — APPROVED** [review]

**Pull request merged**